corehq/apps/reports/analytics/esaccessors.py

```diff
@@ -623,3 +624,9 @@ def get_case_types_for_domain_es(domain, use_case_search=False):
 
 def get_case_search_types_for_domain_es(domain):
     return get_case_types_for_domain_es(domain, True)
+
+
+def get_case_types_for_domain(domain):
+    es_types = get_case_types_for_domain_es(domain)
```

Comment:
What would you think of renaming this to `_get_case_types_for_domain_es` to signal it shouldn't be used? The one wrinkle is it is used in tests, not sure if you have feelings about "private" methods literally only being used in a single module. I don't mind them being imported in tests, but we don't have a team standard.

Reply:
This comment isn't relevant with the newest commit. It might be helpful to have docstrings on the two functions to remind people when they should be used. The variety of ways we have to fetch case types and especially case properties is confusing.

Reply:
Could you rename it something like `get_existing_case_types`, `get_case_types_of_existing_cases` or something more indicative of what it will return? I think knowing what comes out of ES and what comes out of the DD is a bit confusing.

Reply:
Added doc strings to each of the functions to hopefully help with the confusion. Open to renaming as well but want to merge this now to get out in today's deploy. Can rename if necessary in a follow up because I would want a full build to run on that.